lib/player.rb

```diff
@@ -0,0 +1,70 @@
+require_relative '../lib/scoring'
+require_relative '../lib/tile-bag'
+require 'pry'
+
+
+module Scrabble
+
+  class Player
+
+    attr_accessor :name, :plays, :won, :total_score, :tile_bag, :tiles
+
+    def initialize(name = "No Face")
+      # tile_bag = Scrabble::TileBag.new.tile_bag
+      #@tile_bag = tile_bag
+      @name = name
+      @plays = []
+      @won = false
+      @total_score = 0
+      @tiles = []
+
+
+    end
+
+    def play(word) #checks if won and updates current word score
+
+      if @total_score > 100
+        @won = true
```

Comment:
This means I have to **try** to `play` a word **after** I've won to set `@won` to true.